### ISSUE TITLE
Full downloader for Windows

### ISSUE THREAD

**Issue by moatazali** [th-ch/youtube-music#1834]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

the web downloader for windows gets blocked by windows security features

### Proposed Solution

suggesting to add full downloadable install file .exe

### Alternatives Considered

could also be compressed file

**Comment by JellyBrick** [comment]
It's a false positive. and, can't reproduce on the latest release.

Duplicate of #1830.
